### PR TITLE
Add more back end grid classes

### LINKED ISSUE #1119: Neue CSS-Klassen für "full screen mode"
mit dem neuen "full screen mode" ist die kleinste Aufteilung der Eingabemasken vom 50% immer noch recht breit - die Idee von https://github.com/OMOSde/contao-om-backend/blob/master/FEATURES.md#17-backend-css finden wir vom "MM-Team" sehr gut...

Bitte kurze Rückmeldung, ob sowas in Planung ist oder aus OMO übernommen wird - ansonsten würden wir das für MM in unser CSS + Stylepicker einbauen.

### PR DESCRIPTION
**Pull request by leofeyer**
Implements #1119

### CODE REVIEW

File: core-bundle/contao/dca/tl_member.php [hunk 1]
```diff
@@ -206,13 +206,6 @@
 			'eval'                    => array('maxlength'=>64, 'rgxp'=>'phone', 'decodeEntities'=>true, 'feEditable'=>true, 'feViewable'=>true, 'feGroup'=>'contact', 'tl_class'=>'w50'),
 			'sql'                     => "varchar(64) NOT NULL default ''"
 		),
-		'fax' => array
```
Comment (Toflar) [inline core-bundle/contao/dca/tl_member.php]: Why did you remove this field? That's an unnessecary BC break. Data would get lost.

**Review by Toflar — COMMENTED** [review]
I could approve if it weren't for an (unrelated) field removal.

File: core-bundle/contao/dca/tl_member.php (hunk 1 above)
Comment (leofeyer) [reply core-bundle/contao/dca/tl_member.php]: I must have committed this unintentionally. Will revert.

Comment (leofeyer) [reply core-bundle/contao/dca/tl_member.php]: Reverted in c9ad1f13d3a82d668e890a2ddb2018f473ea8161.

File: core-bundle/contao/themes/flexible/basic.css [hunk 2]
```diff
@@ -813,12 +813,16 @@ a.tl_submit {
 	display: table;
 }
 
-.w50 {
+.w25, .w33, .w50, .w66, .w75 {
```
Comment (ausi) [inline core-bundle/contao/themes/flexible/basic.css:816]: I think `.w66` and `.w75` should fall back to 100% width in non-large screens.

```suggestion
.w25, .w33, .w50 {
```

Comment (leofeyer) [reply core-bundle/contao/themes/flexible/basic.css:816]: No, that will not work (see the first form example). The fields have to behave backwards compatible.

Comment (leofeyer) [reply core-bundle/contao/themes/flexible/basic.css:816]: With 100% width, it would fall-back to this:



However, this is how it looks in Contao < 5.1:

**Review by ausi — APPROVED** [review]
